lib/hamster/hash.rb

```diff
@@ -590,6 +590,27 @@ def fetch_values(*wanted)
       Vector.new(array.freeze)
     end
 
+    # Return the value of successively indexing into a hash
+    # If any of the keys is not present in the hash, return `hil`
+    #
+    # @example
+    #   h = Hamster::Hash[:a => 9, :b => Hamster::Hash[:c => 'a', :d => 4], :e => nil]
+    #   h.dig(:b, :c) # => "a"
+    #   h.dig(:b, :f) # => nil
+    # @params keys to fetch from the hash
+    # @return value
```

Comment:
The rest of the Hamster code uses `@return [Object]` in such a case; I'd try to follow suit.